homeassistant/components/synology_dsm/__init__.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, hass: HomeAssistantType, entry: ConfigEntry):
         self.information: SynoDSMInformation = None
         self.utilisation: SynoCoreUtilization = None
         self.storage: SynoStorage = None
+        self.network: SynoDsmNetwork = None
```

Comment:
`SynoDSMNetwork` and not imported.

Can you put it between information and utilisation for naming soting ?